mllib/src/test/scala/org/apache/spark/ml/regression/LinearRegressionSuite.scala

```diff
@@ -1014,7 +1014,7 @@ class LinearRegressionSuite
       checkModelData)
   }
 
-  test("should support all NumericType labels and not support other types") {
+  test("should support all NumericType labels and weights and not support other types") {
```

Comment:
yes, `1e-6` is reasonable as a default value. For some special algo, this may be changed by its author.